config/routes.rb

```diff
@@ -2,5 +2,13 @@
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍